File: timepassword.py
from datetime import datetime

username = "admin"
password = str(int(datetime.now().strftime("%d%m%H%M")))

signed = False
attempts = 3;
while(signed == False):

	username = input("User name: ")
	passwd = input("Password: ")
	print(password)
	if(username == "admin" and passwd == password):
		print("You signed in")
		signed = True
	else:
		attempts = attempts - 1
		if(attempts == 0):
			print("Logging failed.")
			break;
		print("Wrong user name or password! ",attempts," attempts remaining.  Try again:")

File: timepassword.java
import java.util.Date;
import java.text.*;
import java.util.Scanner;

public class timepassword {

	public static void main(String[] args) {

		String username = "admin";
		String password;
		boolean signed = false;
		int loginAttempts = 3;
		Scanner input = new Scanner(System.in);

		Date currentDate = new Date();
		SimpleDateFormat dateFormat = new SimpleDateFormat ("ddMMhhmm");

		password = dateFormat.format(currentDate);

		while(!signed) {

			System.out.print("User name: ");
			String inputUserName = input.next();
			System.out.print("Password: ");
			String inputPasswd = input.next();

			if( username.equals(inputUserName) && password.equals(inputPasswd) ) {
				System.out.println("You signed in.");
				signed = true;
			}

			else {
				loginAttempts--;

				if(loginAttempts == 0) {
					System.out.println("Logging failed.");
					break;
				}

				System.out.println("Wrong user name or password! " + loginAttempts + " attempt remaining." +
				"Try again: ");
			}

		}

	}

}
